### ISSUE TITLE
TypeError: Cannot set property graph of #<Quad> which has only a getter

### ISSUE THREAD

**Issue by yhdelgado** [LinkedDataFragments/Server.js#137]
I am trying to deploy ldf-server in my shared hosting using the latest version of the package. After executed ldf-server config.json everything is ok, but after the first request to the server, the process exit and throw the following error:
```
@ldf/core/lib/datasources/Datasource.js:150
      quad.graph = quad.graph && quad.graph.termType !== 'DefaultGraph' ? quad.graph : (graph || quad.graph);
                 ^
TypeError: Cannot set property graph of #<Quad> which has only a getter
    at SimpleTransformIterator._map
```
To reproduce the error:

You can reproduce the problem by trying the following.
1.  `npm install -g @ldf/server`
2. `ldf-server config.json`

Here my config.json file:
```
{
  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@ldf/server/^3.0.0/components/context.jsonld",
  "@id": "urn:ldf-server:my",
  "import": "preset-qpf:config-defaults.json",

  "title": "Linked Data Fragments server",
  "port": 30000,
  "workers": 1,
  "protocol": "http",

  "datasources": [
    {
      "@id": "ex:linkedDlsource",
      "@type": "HdtDatasource",
      "datasourceTitle": "LinkedDL 2021",
      "description": "Short description here",
      "datasourcePath": "linkeddl",
      "hdtFile": "data/raw-data.hdt"
    }
 ]
}
```
*Note:* I downloaded the source code from github.com and installed it. Everything works as expected. Same HDT file and config file. For some reason, compiling from source code works fine, but from the npm registry doesn't.

**Comment by rubensworks** [comment]
Thanks for reporting.

This looks like a bug because of illegal quad modifications (whereas they are to be considered immutable): https://github.com/LinkedDataFragments/Server.js/blob/master/packages/core/lib/datasources/Datasource.js#L141-L150

@Dexagod I think you wrote this code, could you look into this one?

**Comment by rubensworks** [comment]
Fix released as 3.2.1.
